sources/web/datalab/polymer/test/file-browser-test.ts

```diff
@@ -37,10 +37,13 @@ describe('<file-browser>', () => {
     ApiManagerFactory.getInstance().getBasePath = () => {
       return Promise.resolve('');
     };
-    FileManagerFactory.getInstance().list = (path: string) => {
-      assert(path === startuppath, 'listFilesAsync should be called with the startup path');
-      return Promise.resolve(mockFiles);
-    };
+    const mockFileManager = {
```

Comment:
By doing this, the test no longer calls any of the real code in FileManagerFactory.getInstance. The way it was previously done, it called getInstance(), which instantiated a jupyter-file-manager, then overwrote a method in that file manager.